sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/UnitTestHelper.cs

```diff
@@ -29,34 +29,49 @@ namespace SonarAnalyzer.Helpers
 {
     internal static class UnitTestHelper
     {
-        private static readonly ISet<KnownType> KnownTestMethodAttributes = new HashSet<KnownType>
+        public static readonly IEnumerable<KnownType> KnownTestMethodAttributes_MSTest = new HashSet<KnownType>
```

Comment:
Changed to _IEnumerable_ rather than _ISet_ since the field is now publicly visible (otherwise we get "use an immutable type" issues).